package.json

```diff
@@ -148,6 +148,23 @@
                 "description": "%java.debugger.launch.console.description%",
                 "default": "internalConsole"
               },
+              "shortenCommandLine": {
+                "type": "string",
+                "enum": [
+                  "auto",
```

Comment:
sequence: none should be put above auto

Reply:
Is there any explanation why need to sort the sequence like this?

Reply:
none is consistent with the original semantic